examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -84,6 +90,9 @@ static struct uip_icmp6_echo_reply_notification echo_reply_notification;
 static struct etimer echo_request_timer;
 int def_rt_rssi = 0;
 #endif
+
+uint16_t singleSample;
```

Comment:
This variable probably should be declared static and renamed to something along the lines of single_adc_sample